addons/web_editor/static/lib/odoo-editor/src/OdooEditor.js

```diff
@@ -1535,13 +1535,17 @@ export class OdooEditor extends EventTarget {
      * @returns {?}
      */
     _applyRawCommand(method, ...args) {
-        const sel = this.document.getSelection();
+        let sel = this.document.getSelection();
         if (
             !this.editable.contains(sel.anchorNode) ||
             (sel.anchorNode !== sel.focusNode && !this.editable.contains(sel.focusNode))
         ) {
             // Do not apply commands out of the editable area.
-            return false;
+            if (this._latestComputedSelectionInEditable) {
+                sel = this._latestComputedSelectionInEditable;
```

Comment:
I'm confused about this one. I guess it's the one you referred to when you said you were surprised we didn't do that all the time... :/

Reply:
Yes, that one and the previous one. What confuses you exactly ?